kloppy/domain/models/pitch.py

```diff
@@ -25,3 +24,7 @@ class PitchDimensions:
 class Point:
     x: float
     y: float
+
+    def distance_to_point(self, Point) -> int:
```

Comment:
your definition of the argument is a bit incorrect and the return type should be float.
`def distance_to_point(self, point: Point) -> float"

and then do something with `self.x, self.y, point.x and point.y`. Hint: no zip needed. Your construction is only needed when you don't know the number of dimensions.